### ISSUE TITLE
Write backup summary in git commit message

### ISSUE THREAD

**Issue by jevonearth** [ecadlabs/rosdump#3]
For each backup run, backups may succeed or fail. To make this information visable overtime, I propose we write a backup summary to the commit log.

The `rosdump` utility can write a summary like follows:

```
Routers backup 4 devices of 5 devices

OK 10.60.99.2
OK 10.60.99.3
OK 10.60.99.1
FAIL 10.60.99.4
OK 10.60.58.1
```

and use this summary as the commit message when using the git storage back-end.

What about when no changes are detected?

We push an empty commit, to make clear that a change happened;

The git cli option for this is:

`git commit --allow-empty`

**Comment by jevonearth** [comment]
Tested and released in v0.1.1 
Looks great @e-asphyx

**Comment by laurivosandi** [comment]
I don't think empty commits is the way to go here, it would make more sense to export Prometheus metric which would alert if some of the targets were unreachable...
